datafusion/expr/src/udf.rs

```diff
@@ -785,6 +793,17 @@ pub trait ScalarUDFImpl: Debug + DynEq + DynHash + Send + Sync {
         Ok(Some(vec![]))
     }
 
+    /// Propagate set statistics from children through this function.
+    ///
+    /// Returns `Ok(None)` by default, indicating the function does not support
+    /// set-stat based pruning.
+    fn propagate_set_stats(
+        &self,
+        _child_set_stats: &[SetStats],
```

Comment:
```suggestion
        _arg_set_stats: &[SetStats],
```

since we tend to call them arguments for functions not children?